annals-of-botany.csl

```diff
@@ -104,14 +104,25 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <group suffix=". ">
+            <text macro="title" font-style="italic" prefix=" "/>
+          </group>
```

Comment:
@damnation333, while it's good to put as much separating punctuation in groups as possible, groups with only a single child element only complicate things, and should be avoided. (moving the suffix from the group to the `text` element would allow you to delete lines 108 and 110)